packaging/flavorgen/flavors/generators.go

```diff
@@ -331,6 +341,105 @@ func defaultPreKubeadmCommands() []string {
 	}
 }
 
+func kubeVIPPod() string {
+	hostPathType := v1.HostPathFileOrCreate
+	pod := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       typeToKind(&v1.Pod{}),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kube-vip",
+			Namespace: "kube-system",
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:  "kube-vip",
+					Image: "plndr/kube-vip:0.1.6",
```

Comment:
Either here or a follow-up: let's allow the image to be customizable + defaulted if the user doesn't specify an alternative.

Reply:
we don't enable custom images in our upstream templates (e.g. CCM and CSI), usually users have template overrides with their own images

Reply:
👍